<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BondOrderPopup</class>
 <widget class="QWidget" name="BondOrderPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>73</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QToolButton" name="triple_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton { border : none; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="sketcher.qrc">
       <normaloff>:/icons/triple_bond.png</normaloff>:/icons/triple_bond.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="double_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton { border : none; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="sketcher.qrc">
       <normaloff>:/icons/double_bond.png</normaloff>:/icons/double_bond.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="coordinate_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton { border : none; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="sketcher.qrc">
       <normaloff>:/icons/coordinate_bond.png</normaloff>:/icons/coordinate_bond.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="group"/>
 </buttongroups>
</ui>
